src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,17 @@
-export const CommentInfo = () => <>Put the comment here</>;
+export const CommentInfo = ({ comment }) => (
```

Comment:
```suggestion
export const CommentInfo = ({ comment: { name, ... } }) => (
```
Consider using restructuring here